Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/SpeechInputHandler.cs

```diff
@@ -108,6 +138,16 @@ void IMixedRealitySpeechHandler.OnSpeechKeywordRecognized(SpeechEventData eventD
             {
                 keywordResponse.Invoke();
                 eventData.Use();
+
+                // OS will display the tooltip for the "Select" keyword. 
+                if ((speechConfirmationTooltipPrefabInstance != null) && (eventData.Command.Keyword.ToLower() != "select"))
+                {
+                    // Update the text label with recognized keyword
+                    speechConfirmationTooltipPrefabInstance.GetComponentInChildren<TextMeshPro>().text = eventData.Command.Keyword;
+                    // Trigger animation of the Speech Confirmation Tooltip prefab
+                    speechConfirmationTooltipPrefabInstance.GetComponent<Animator>().SetTrigger("Confirmed");
```

Comment:
Can you please confirm that the `speechConfirmationTooltipPrefabInstance` deletes itself when its animation is over? Don't want to leave orphaned / old speechConfirmationTooltipPrefabInstance instances all over the scene.

Reply:
Updated script to instantiate on recognition & destroyed on animation complete